drivers/hmis/spec/system/hmis/assessment_definitions_spec.rb

```diff
@@ -124,55 +175,120 @@
       click_link 'Intake'
     end
 
+    def select_member(client)
+      find('button[role="tab"]', text: client.brief_name).click
+      expect(find('button[role="tab"][aria-selected="true"]', text: client.brief_name)).to be_present
+    end
+
+    def unlock_household_assessment
+      click_button 'Unlock Assessment'
+      assert_no_text 'This assessment has been submitted.' # Unlock succeeded
+    end
+
+    def submit_household_assessment
+      click_button 'Save & Submit'
+      assert_text 'This assessment has been submitted' # Submit succeeded
+    end
+
+    def save_household_assessment
+      click_button 'Save Assessment'
+      assert_text(/Last saved [0-9] seconds? ago/) # Save succeeded
+    end
```

Comment:
Should these go in `spec/support/e2e_setup.rb`? I could see them being useful in other tests. Not sure if that class was meant to hold more general MUI actions, and maybe these should go in a separate class. Any thoughts on the convention there @ttoomey?